src/components/Result.jsx

```diff
@@ -1,18 +1,22 @@
-import React, { Component } from 'react';
+import { React } from 'react';
+import PropTypes from 'prop-types';
 
-class Result extends Component {
-  constructor() {
-    super();
-    this.state = {};
-  }
+const Result = ({ total, next, operation }) => (
+  <div className="result-container">
+    {console.log(total, next, operation)}
+    <input className="result-input" type="text" readOnly value={total || next || operation || 0} />
+  </div>
+);
```

Comment:
- [ ] It would be nice if you fix the issue of the next operation on the screen so that when I enter an operation means (+, -, *, /), and then I enter the second number then you can see the both are not visible on the screen which is not according to the current functionality standards of moderns calculators 🤔 Kindly make the next and operation also visible on the screen to make your app professional 👍